From: Tzafrir Cohen <tzafrir@debian.org>
Subject: keys for Fedora 25
Forwarded: no

Keys were not included in the mock distribution. Taken from
the distribution-gpg-keys package.

--- /dev/null
+++ b/etc/pki/RPM-GPG-KEY-fedora-25-primary
@@ -0,0 +1,32 @@
+pub  4096R/FDB19C98 2016-03-31 Fedora 25 Primary (25) <fedora-25-primary@fedoraproject.org>
+
+-----BEGIN PGP PUBLIC KEY BLOCK-----
+Version: GnuPG v1
+
+mQINBFb9YzMBEACy1RmbMa6MNIpfHYxLwgCgBVnFYCdCHZqWfYYYK14potfJ9uI2
+4Y4w+oHiLeZ/HoG1EBQiDfXHetGZECAKEYQlE7BbRBcd3An9GalKTkWzcshhHFx7
+f5JIprL0uY8x2D9HmCfAjMxoh6usWjmAQ+DUYd48iYCkahyZa0/2CgX9HIcEz/M/
+oDeQbTwzw9AQbQz382oOErfRaXE/DQrjlx2ln0iejidiOe7DzGZOH9/Foc2KN062
+A9VnZ7tU1ACKT8NxZ78RaBL3qmvMGdb7kf7GywjpRNo4J7XCQUP+nP51eCur2wMS
+4mY2idDL8Ojouta79pPrviVLmwzunJoFnBcnIhbndebdxPqgOA5XAOaTdLtgurMq
+90V45DPyJpkdEyptovksH7zYNGEIGB8cFmrVgUwriB0TLNJTEcM4Knbh4imfTX42
+vCE+rEHn3YVqubG7rggibKznJbflwQcqOYZHLlPGYCxO47aaFUo5qJN7QN3lxajb
+SzL/SdoHrVL67unzmHyktx5uF8Fv6EDgUV6NCb/IBiEwhR8YHi86NQ8nsI3K8Zhv
+EnIxghJQD+cn3ykthwqYmZwi2PJDBiZsOGf3iXbalAjU3JVqoA7mboRPR+IBXQxK
+xvAEpyIGeSUN8yBn+JVDRwZ37kkUVs2AOeUwMlnfFSqYFfmqbeQ73A9ECwARAQAB
+tDxGZWRvcmEgMjUgUHJpbWFyeSAoMjUpIDxmZWRvcmEtMjUtcHJpbWFyeUBmZWRv
+cmFwcm9qZWN0Lm9yZz6JAjgEEwECACIFAlb9YzMCGw8GCwkIBwMCBhUIAgkKCwQW
+AgMBAh4BAheAAAoJEECJ2PL9sZyY1TEP/0u/v4g8HEdl9gqlhV179vXCJJiGtzB0
+7IGAu++mrsxBrDpqPZTEs6dG5MyzvhhHcmHYrZIiicPAeL9xlZ75oIqQuvjDncoM
+kROSGvtfUnvocZhQIPvvkgWe3UAmmP3cSlVzu3KtbTpM+KL71incWo4Tentq9L/f
+vsow7vvGbKUMoSSZbAMfjJkzlzSDNlFtaRkrCBQFJ76EKeggjnEZ8H0cowCdGuyv
+uBoxQeeQM13b2T9c/uyrXCIcasaOTIKTcqTjbJUTIC2NIZ8OHjtlxZacEaN3ml1M
+lNRtbIvqzbtv+sb+DsOVTyd1XIcxU9s+TDKvUm0OBNvj3Bm2BQbi8RHyLFbHWvhx
+Gjzb8Wb/MnlcdTlk3M2iPv8dWHXjEM9n9TKyStdpBD9X3P/Gy2gUquHgkl8p+r8o
+xmzNH534mKH47kPL/trKInKwv0fkBwxvuPgHG0n79eMHQenVA8gXzG4P6JkcyObA
+6xGEEQ/wXFF0gLksmwFWuPm2GcnOI5KmGNgDP2PMhS8/cfJfW04a/tL2T2zr4CmE
+LynbOvY/yOJk7/2W3Cb47+yhqo/htrpJDP6n6zQNNk3+e8EVgfhkQqFxom8yCmEP
+pW0gBFeE83VoytYPXRkavwmFR+tplyZfOkXG9gysTn8SpRp5+B44O+VeaZumanQZ
+kRFmBygMR6M/
+=NrXo
+-----END PGP PUBLIC KEY BLOCK-----
--- /dev/null
+++ b/etc/pki/RPM-GPG-KEY-fedora-25-secondary
@@ -0,0 +1,55 @@
+pub  4096R/E372E838 2016-03-31 Fedora 25 Secondary (25) <fedora-25-secondary@fedoraproject.org>
+sub  2048g/A0582CC4 2016-03-31
+
+-----BEGIN PGP PUBLIC KEY BLOCK-----
+Version: GnuPG v1
+
+mQINBFb9eUMBEAC4XFMbnbuiLmUPRqenm9SRcNkrXWvp33iBJBUKfqU2pKApkpDS
+YvLNukZlWDnwlyrXzPB8WJQOcugbtmN7nWK1vjAJnrT9BHPBnLZvsZoYuBnEvzsl
+CbV6cOOal85LFtdqpu5Pir9OD5OSgseubRh21tsuM5uNPOE8pL1ffsRt6zD0paf0
+G1gr5+9rSc2cZ53RgwYBAJV4tethe+RDGSs/6Lrv6v8SkkgSW4USt7R73govRLTU
+XRW6NB5s5S74EiApyAqgC2UMNQZw6PjAGUFw8BFhJcUNzVnIzGrXdMGdBQ1G12io
+5F3PDsqHZ9V3qqLxUgNhlK3W/HvIRv0irRviSOwux8QnE3toSqBaWbS445L9Ar2z
+7SHdEb8InNdV9dcPL1Y5Nes8Do5Sgz6syBVwk4vfJMAs6w7Gr/fdS5OdOCkv2XrX
+f4/Bmxc5GD0KgXTduErcnWCoswJx3CVbUt/a5/DozQH6wxjlxDfVE1Y9DkzxPeoe
+J8nMzidf6N1YkiaF5fRIoUY0edYh4ID4jT/NTRTAAc17BaftK3prqc+x0MWROuiT
+W2Y8tEIZthNT6jBJsVlV9ip0JWGAM4cHScsnuXnoZqFQdpe3En2kcxcWb7t/Spwc
+cMLfrD4YJw4TVRN7qSk/2XvNmM0C1gNpBSGrjXI9JGOY/SBScUx5Mz/9IwARAQAB
+tEBGZWRvcmEgMjUgU2Vjb25kYXJ5ICgyNSkgPGZlZG9yYS0yNS1zZWNvbmRhcnlA
+ZmVkb3JhcHJvamVjdC5vcmc+iQI4BBMBAgAiBQJW/XlDAhsPBgsJCAcDAgYVCAIJ
+CgsEFgIDAQIeAQIXgAAKCRAaGFzd43LoOOxpD/0XWzF+OvKyhM7xz19F6HG6QKTx
+3G6gxnqzrOb9XVB805C7zPEmhURO6TxxZBigwothmjovHMhVrdeVtChJ+UWVy16W
+fpemCwiZ4saLfyMVQQBdovNTwTLAawpp0frkGqO8iGAuEdnCSrTZKZLjg9xiTJyz
+C5slpje5cHEHA51pUUuoK542ne2x+08KMhJ60Ppo8seSRHQSbgnqLSIo9Q3QnA5H
+aRHjb+nb4cvXHsLRqoQbNcjMf9SGe7tDhLzUjeGG+J6b3H3evNp50pMTg610y47U
+rpy5bXjHOEJSx8zFjfXo3SEKFCduDkZUIuBCw8YtjtaRSM2ucym6WZr7Veu6E0RX
+h4xoB9d7kp6gHXfaia9BRlTM4AuLhvlU3CSQycZllrZhW59Pm4DEyHC3aoQZnP7a
+ypPSHlUE16nzTkdPLa/Om9FlyXH2lnwV/dQ3uNZeb5H+wZBoPTpcmgfCSRbZCIxF
+LTlUuZ/LVGN2ZfDtemeNI/Myf/fiGUU4Mc4+PISwvRVvZLJP1W8y4ERcmtK3khG7
+lkU9NgSMDzDvkP/EmXBldciqlTykYAzKw8geShmgL2Tc4bTvqJxDMcDHLhYz0R0o
+KHqpjsRpJt3a0y+Z+15gG1winnoOaIxF6QqXCOWc2M0mIPmazkOXWITQEhFMn/GV
+rYtB2w4+46rQ22pp0LkCDQRW/XlDEAgAhRX7+JqpohxxLest1GrtxWjEwDrUy5Jr
+2fa/3l0wgvQpooHd+RK31m0CvTC5udFz6ysx+L8LC5yWQxHiR5VNSYy9uNfaM8Kf
+BklM2Q2SRZlv/BlKN9Djec5l17aldm/6zgL6/gdx6TkBBdv0SQf5fb/mm+j+ot3g
+LPveEiXUb/0pTiWjEAP/N8yNePctr9JX4M6maTtQq8iH/o6H2byj87wicgr4gaI9
+RDe0F6BgAJ1IVCBt1RB19fPt5Z8BgcNhZdxi0xzn65uYg0lQ5yjwBSHP4qAdLRAS
+aerEhdA0gBxf4Gj1by8fPpaWKSZ3VFcVIP2loqXkm/Uco2+h7AUgvwADBQf+PV5o
+CcgExnG03MMMnCc75vP4hDK+EyOtAWik4WJBuCdNoIoCvysM+GBSXt7XD9FO3qkn
+UEQlWDQMeWZ+NBIQj7tk5WWz9/EZAVdj4DeN3VsbHpgIbOAHwvmNUZjCLvpBE3+K
+FtFZOffX629n8E2YhnLZHHIu6TbPesPPPdqpJbpU7L56nDO+ZqcqMY8rIOeViZYG
+zUBdH4JbwGmq2i9W8tf497CjdWN81QO4li3s688fSj3hcz0ewj+o8WDW/0yy0NkX
+lw7lO2TKownpnLJlcm2aJcptaN7MbmfvEg9mAMdSjd0CFoTSUuzF7/m+C8TyQTPE
+PsE74vH4MHuL7rANpIkCHwQYAQIACQUCVv15QwIbDAAKCRAaGFzd43LoOC1nD/96
+jEXAmjTIvwRf9b9CI+S63JDUTi0WyZaXMEzwfnmQY0KXDwbXuidskbpS3DmCcLjF
+kRdWtiWdzRp6aCT95GxP+kRz6Zy7PkjpPPZQ0Tj+b3zInkOQlodPSTfWUbDztaHw
+zzx5g3L9YebmWULOZvV+rqAWseUw8xJ8VygDnKNBvXM4berDt9B8A74IlCJSvGpf
+zL+UwrCAOJ9etYpJjfqRN+TW3k3COrYqVN7R8Cq42J4sFOE9BhSr26NdTDT3wCrV
+veDRJ20hChETg3mhzTcYj3JIbwEUO2hKVUB+bkOjmCJGYV/sJM87cUdi0D1OacqP
+EbKNCbMidppBcNG1aGY310S4uVB0Wyy/UoqJH9szyIBkT7eWaNthKcd7UnlVRaM1
+vv5sy5ZudJmTpNuupKfpQEJPnRM2uVBm5Rgpa3tawzUDOFM4uM5z/vyZ6zYTkstv
+VN55DlgyUSx5oImtHcYiVGYE2P3pCnxMAFzmxUeIwipva0k2FYf10eavFqvprWHY
+UYkjramXyyDli2lmI5CEWLL3ZUgCzDwWSMxkTdaBGoY5XLVPl+DNHYG2G6F9YqnI
+nmQndpeUwSDspU9Nv/VeSnn+cCLWaJuqpNLNoaxju14OtP5F6GcrKsphlD2zjB9+
+KgCdlumPwrKe3KQBUOKYTnv8MMfulZHueQvy51IDZQ==
+=fWOn
+-----END PGP PUBLIC KEY BLOCK-----
